extensions/spotify-controls/src/privateSession.ts

```diff
@@ -0,0 +1,26 @@
+import { showToast, Toast } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+import { buildScriptEnsuringSpotifyIsRunning, runAppleScriptSilently } from "./utils";
+
+export default async () => {
+  const script = buildScriptEnsuringSpotifyIsRunning(`
+  tell application "System Events"
+	  tell process "Spotify"
+    click menu item "Private Session" of menu 1 of menu bar item "Spotify" of menu bar 1
+    set privateSession to value of attribute "AXMenuItemMarkChar" of menu item "Private Session" of menu 1 of menu bar item "Spotify" of menu bar 1
+	  end tell
```

Comment:
Could this code fail for non-english languages? Not sure if Spotify localizes this menu item. Maybe it's better to rely on the position of the menu item? I suspect this won't change?!

Reply:
Not sure about non english systems! Changed it to use the position of the menu item as you suggested just in case.

Reply:
Works, tested with the Danish language 😊